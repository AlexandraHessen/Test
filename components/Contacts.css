.Contacts{
    font-weight: bold;

}

.Contacts li{
    list-style-type: none;
}

