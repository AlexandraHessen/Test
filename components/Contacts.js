import React from 'react';

import './Contacts.css';

class Contacts extends React.Component{
    render(){
        return (
            <ul className="Contacts">
                <li>+375 (29) 844-44-88</li>
                <li>+375 (44) 888-44-88</li>
                <li>Пн-Пт: 9:00-18:00</li>
                <li>г.Минск, пр.Победителей, 65</li>
            </ul>
        )
    }
}

export default Contacts;