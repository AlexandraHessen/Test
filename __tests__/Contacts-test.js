"use strict";

import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

import Contacts from '../components/Contacts';
                    //импортируем файл который будем тестировать

test('Contacts', () => {
                            // создаём тестовую версию компонента который будем тестировать
  const component = renderer.create(
    <BrowserRouter>
      <Contacts />
    </BrowserRouter>
  );

                            // получаем снэпшот (HTML-снимок) компонента для сверки, чтобы вёрстка не испортилась
  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

    
});
