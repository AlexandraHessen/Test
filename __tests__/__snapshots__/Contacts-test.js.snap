// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts 1`] = `
<ul
  className="Contacts"
>
  <li>
    +375 (29) 844-44-88
  </li>
  <li>
    +375 (44) 888-44-88
  </li>
  <li>
    Пн-Пт: 9:00-18:00
  </li>
  <li>
    г.Минск, пр.Победителей, 65
  </li>
</ul>
`;
